src/modules/gnd_pos_control/GroundRoverPositionControl.cpp

```diff
@@ -135,11 +124,7 @@ GroundRoverPositionControl::GroundRoverPositionControl() :
 	_parameter_handles.l1_damping = param_find("GND_L1_DAMPING");
 	_parameter_handles.l1_distance = param_find("GND_L1_DIST");
 
-
-	_parameter_handles.airspeed_min = param_find("FW_AIRSPD_MIN");
-	_parameter_handles.airspeed_trim = param_find("FW_AIRSPD_TRIM");
 	_parameter_handles.airspeed_max = param_find("FW_AIRSPD_MAX");
```

Comment:
A proper mount in the side of the vehicle will do the trick I suppose.
https://cloud.githubusercontent.com/assets/11164269/25453855/c4fb02a0-2aca-11e7-8c44-d17cf2ca19ef.jpg
I think it has enough height to mount the device on its side I suppose. The other way would be what I said, but that requires a companion computer.